windows_impl.go

```diff
@@ -0,0 +1,1670 @@
+// Copyright 2017 The Virtual Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// THIS FILE IS AUTOGENERATED BY generator.go
```

Comment:
See ["standard way to recognize machine-generated files"](https://github.com/golang/go/issues/13560#issuecomment-277804473):

> The text must appear as the first line of a properly formatted Go // comment, and that comment must appear before but not be attached to the package clause and before any /* */ comment. This is similar to the rules for build tags.
>
> The comment line must match the case-sensitive regular expression (in Go syntax):
>
> `^// Code generated .* DO NOT EDIT\.$`
>
> The .* means the tool can put whatever folderol it wants in there, but the comment must be a single line and must start with Code generated and end with DO NOT EDIT., with a period.